ExampleApp/equityTicketConfig.json

```diff
@@ -0,0 +1,173 @@
+                "label": "Price",
+                "inputType": "options",
+                "options":[
+                           {
+                           "value":"market",
+                           "label": "Market"
+                           },
+                           {
+                           "value":"limit",
+                           "label": "Limit"
+                           },
+                           {
+                           "value":"stopLimit",
+                           "label": "Stop Limit"
+                           },
+                           {
+                           "value":"stop",
+                           "label": "Stop"
+                           }
+                           ]
+                },
+                {
+                "apiParameter": "orderQuantityType",
+                "label": "Quantity Type",
+                "inputType": "options",
+                "options":[
+                           {
+                           "value":"SHARES",
+                           "label":"Shares"
+                           },
+                           {
+                           "value":"TOTAL_VALUE",
+                           "label": "USD",
+                           "dependencies":[
+                                            {
+                                            "condition": "==",
+                                            "value": "buy",
+                                            "criteria":"orderAction"
+                                            },
+                                            {
+                                            "condition": "==",
+                                            "value": "market",
+                                            "criteria":"orderPrice"
+                                            },
+                                            {
+                                            "condition": "==",
+                                            "value": "buy",
+                                            "criteria":"orderAction"
+                                            },
+                                            {
+                                            "condition": "==",
+                                            "value": "market",
+                                            "criteria":"orderPrice"
+                                            }
+                                           ]
+                           }
+                           ]
+                },
+                {
+                "apiParameter": "orderQuantity",
+                "label": "Quantity",
+                "inputType": "decimal"
+                },
+                {
+                "apiParameter": "orderLimit",
+                "label": "Limit",
+                "inputType": "decimal",
+                "dependencies": [
+                {
+                                 "condition": "==",
```

Comment:
`condition` seems like it would be better named as `operator` -- as I see it the entirety of the statement is the `condition`, not the type of operator being used.